### PR TITLE
closing the files when we're done reading them

### PR DESCRIPTION
**Pull request by kay0u**

### CODE REVIEW

**Review by Josue-T — APPROVED** [review]

File: config.lua [hunk 1]
```diff
@@ -11,14 +11,16 @@ function get_config()
     -- Load the configuration file
     local conf_file = assert(io.open(conf_path, "r"), "Configuration file is missing")
     local conf = json.decode(conf_file:read("*all"))
-
+    conf_file:close()
```
Comment (kay0u) [inline config.lua]: I should check if con_file is not `nil`

Comment (alexAubin) [reply config.lua]: eeeh bumpity bump ? apart from this comment this PR looks like easy to merge?

Comment (kay0u) [reply config.lua]:
```suggestion
    if conf_file ~= nil then
        conf_file:close()
    end
```

Comment (kay0u) [reply config.lua]: Indeed, I think it should works™ ( suggestion not tested)

**Comment by alexAubin** [comment]
Yolomergin after applying Kayou's suggestion

**Pull request merged**